routes/api.php

```diff
@@ -14,9 +14,12 @@
 | is assigned the "api" middleware group. Enjoy building your API!
 |
 */
+Route::post('/admin/register', [AdminAuthController::class, 'register']);
+Route::post('/admin/login', [AdminAuthController::class, 'login']);
 
-Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
-    return $request->user();
+Route::group(['middleware' => ['auth:sanctum', 'role:admin']], function () {
+    Route::post('store/category', [CategoryController::class, 'store']);
+});
+Route::group(['middleware' => ['auth:sanctum']], function () {
+    Route::post('/admin/logout', [AdminAuthController::class, 'logout']);
 });
```

Comment:
1. Route `store/category` is not restful `categories` with post method is enough to describe the endpoint
2. Why Applying role:admin on category creation, Role & permissions is for abilities and we should have multiple admins with multiple roles so in that case we should think about roles & permissions context before assigning them